Jenkinsfile

```diff
@@ -205,6 +249,7 @@ pipeline {
                 }
             }
         }
+
```

Comment:
How about adding a small comment about the 'Stage - OpenShift Template' and 'Rollout' too? Maybe a minimal reference their Dev counterparts? If only for completeness...